service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GpcException.java

```diff
@@ -0,0 +1,8 @@
+package uk.nhs.adaptors.gp2gp.gpc;
+
+public class GpcException extends RuntimeException {
```

Comment:
I saw another similar RTE somewhere. How about a single type for all GPC-related errors: `GpConnectException`